site/migration/01-migration.md

```diff
@@ -0,0 +1,257 @@
+## Migration from kpt `v0.39.0` to kpt `v1.0`
```

Comment:
It may be better to have this document live in `docs/` instead of `site/`. We can link to this doc from installation page and the fix command instead of having a new top-level section in the navbar.

Reply:
Will do that in next PR.

Reply:
Let's do this in this PR, otherwise the navbar will be affected in the meantime.

Reply:
It is not possible to point to files outside the site folder from the files within site folder. So moved the folder to installation folder and added a link in installation guide. Remove it from sidebar to not pollute the view.